code/game/objects/items/devices/uplinks.dm

```diff
@@ -405,6 +405,12 @@ GLOBAL_LIST_EMPTY(world_uplinks)
 	hidden_uplink = new(src)
 	icon_state = "radio"
 
+/obj/item/radio/uplink/AltClick()
+	return
+
+/obj/item/radio/uplink/CtrlShiftClick()
```

Comment:
This wont stop the examine from showing that can do these things, even though you removed the functionality. This should probably not be a subtype of radio

Reply:
I removed this from examine in obj/item/radio, so only the subtypes have it